Jordan_Class.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri May 22 18:15:13 2020
+
+@author: User
+"""
+import re # importing regular expression operations, which we need later to access the libraries
+import random # later we want to pick a chatbot answer to a statement by random
+from Jordan_Libraries import library_smalltalk, library_caring, library_cursing, library_corona, library_meditating
+# and here we imported all the topic libraries
+from playsound import playsound
+      
+      
+class Jordan:
+    """
+    A class containing Jordans skills.
+    
+    ...
+    
+    Attributes 
+    ----------)
+    keys_smalltalk : list
+        Nested lists with strings
+    keys_cursing : list
+        Nested lists with strings
+    keys_caring : list
+        Nested lists with strings
+    keys_corona : list
+        Nested lists with strings
+    
+    Methods (Functions)
+    ----------
+    smalltalk_Jordan(userInput)
+        function that initiates small-talk with Jordan
+    cursing_Jordan(userInput)
+        function that initiates cursing with Jordan
+    caring_Jordan(userInput)
+        function that initiates a talk with Jordan about your sorrows
+    corona_Jordan(userInput)    
+        function that initiates a talk with Jordan about corona
+    """
+    
+    def __init__(self): # this function is always called when a new object of the class is called
+        """
+        This function is to determine attributes, 
+        which can be applied in Jordan's topic functions.
+        
+        """
+        
+        self.keys_smalltalk = list(map(lambda x:re.compile(x[0], re.IGNORECASE), library_smalltalk)) 
+            # list(map()) applies a function to all elements of a specified object, in this case the small-talk library
+            # lambda makes sure that re.compile is applied in a certain way to all elements of the library without being case-sensitive
```

Comment:
Really great explaining of your code!
I'd just recommend breaking your description down into more lines.
(The python guide recommends a maximum of 79 characters, and on this line, for example, you have x1.5 that number.)